.dockerignore

```diff
@@ -2,7 +2,7 @@
 
 **/.DS_Store
 **/.dockerignore
-**/.env
+# **/.env
```

Comment:
Fair point. I've added this since it contains the info for a specific app registration config, which should be configured ahead of deployment as per deployment instructions. Shall we find another way to populate app registration env var ahead of deployment?

Reply:
Is this common in Aspire projects to checkin .env? I find that pretty surprising. In AI land that file is used all over the place for secrets and commonly excluded via .gitignore. I think it is asking for trouble.

Reply:
This is not to check it in the repo, only to add it to the container. Which was needed from my perspective only to handle app registration config at runtime.